pages/maps/index.vue

```diff
@@ -14,18 +23,17 @@
 
 <script>
 export default {
-  async mounted() {
+  async fetch() {
     try {
-      await this.$store.dispatch('map/getAllMaps') 
-    } catch(e) {
-      console.log('whoooa', e)
+      await this.$store.dispatch("map/getAllMaps");
+    } catch (e) {
+      console.log("whoooa", e);
     }
   },
   computed: {
     maps() {
-      console.log('hi im the view', this.$store.state.map.maps)
-      return  this.$store.state.map.maps;
+      return this.$store.state.map.maps;
```

Comment:
Same here, should just mapGetters and make your life easier.

https://vuex.vuejs.org/guide/getters.html#the-mapgetters-helper